.github/actions/build_binaries/action.yml

```diff
@@ -0,0 +1,55 @@
+name: Build Ockam Binaries For Different Architechtures
+description: Build Ockam Binaries For Different Architechtures
+
+inputs:
+  use_cross_build:
+    description: Indicate If Cross Should Be Used To Build Binary Instead Of Cargo
```

Comment:
I don't know what the accepted convention but Do We Need To Capitalize Every Word :-)?

Reply:
I don't think there's a convention, its more of preference of mine.